src/placeos-rest-api/utilities/current-user.cr

```diff
@@ -98,12 +98,9 @@ module PlaceOS::Api
     end
 
     def can_scopes_read(*scope_names)
-      can_read = false
-      scope_names.each do |scope_name|
-        can_read = can_scope_read(scope_name)
-        break if can_read
-      end
-      raise Error::Forbidden.new unless can_read
+      raise Error::Forbidden.new unless scope_names.any? do |scope_name|
+                                          can_scope_read(scope_name)
+                                        end
```

Comment:
```suggestion
      raise Error::Forbidden.new unless scope_names.any? { |scope| can_scope_read(scope) }
```

The formatting is a lil weird, maybe it's cleaner to do on a single line

Reply:
That's the format from crystal format, tried with a single line and it fails the style CI